crates/ruma-federation-api/src/authenticated_media/get_content.rs

```diff
@@ -0,0 +1,106 @@
+//! `GET /_matrix/federation/*/media/download/{mediaId}`
+//!
+//! Retrieve content from the media store.
+
+pub mod unstable {
+    //! `/unstable/org.matrix.msc3916/` ([MSC])
+    //!
+    //! [MSC]: https://github.com/matrix-org/matrix-spec-proposals/pull/3916
+
+    use std::time::Duration;
+
+    use ruma_common::{
+        api::{request, Metadata},
+        metadata,
+    };
+
+    use crate::authenticated_media::{ContentMetadata, FileOrLocation};
+
+    const METADATA: Metadata = metadata! {
+        method: GET,
+        rate_limited: false,
+        authentication: AccessToken,
+        history: {
+            unstable => "/_matrix/federation/unstable/org.matrix.msc3916/media/download/:media_id",
```

Comment:
[The unstable endpoint has two versions](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3916-authentication-for-media.md?plain=1#L385-L387).
```suggestion
            unstable => "/_matrix/federation/unstable/org.matrix.msc3916.v2/media/download/:media_id",
```

Reply:
Nice catch!

Reply:
Fixed.